.template/addons/nginx/config/nginx/app.conf.template.tt

```diff
@@ -0,0 +1,113 @@
+worker_processes auto;  ## Default: 1
+
```

Comment:
@andyduong1920 Should we have 2 versions of template, 1 for api and 1 for web? Because api will not need gzip and static file settings?

Reply:
Fixed in https://github.com/nimblehq/rails-templates/pull/232/commits/e770d2c22141fd6c4ffc2a5e450e4b69b854cff5 @ankitkalia1195